bolt-modules/boltlib/spec/functions/apply_prep_spec.rb

```diff
@@ -208,6 +208,48 @@
         expect(inventory.facts(target)).to eq(fact)
       end
     end
+
+    context 'with default plugin inventory v2' do
+      let(:hostname) { 'agentless' }
+      let(:data) {
+        {
+          'version' => 2,
+          'targets' => [{ 'uri' => hostname }]
+        }
+      }
+
+      let(:config) { Bolt::Config.new(Bolt::Boltdir.new('.'), {}) }
+      let(:pal) { nil }
+      let(:plugins) { Bolt::Plugin.new(config, pal, Bolt::Analytics::NoopClient.new) }
+      let(:inventory) { Bolt::Inventory.create_version(data, config, plugins) }
+      let(:target) { inventory.get_target(hostname) }
+      let(:targets) { inventory.get_targets(hostname) }
```

Comment:
I don't think this gets used anywhere?

Reply:
Yes, targets is used in the before each block.